app/javascript/packages/document-capture/services/upload.ts

```diff
@@ -87,6 +87,9 @@ const upload: UploadImplementation = async function (payload, { method = 'POST',
     body: toFormData(payload),
     json: false,
     read: false,
+    headers: {
+      Accept: 'application/json',
+    },
```

Comment:
This code already expected the response to be returned as JSON (see line 101 below), but it sends the request as a multipart form. In order for the new code to detect the request format as JSON, we need to explicitly specify the accept MIME type.

Reply:
will this work with a lowercase `accept:` or is the uppercase necessary?
```suggestion
    headers: {
      accept: 'application/json',
    },
```